Formula/utftex.rb

```diff
@@ -0,0 +1,21 @@
+class Utftex < Formula
+  desc "Pretty print math in monospace fonts, using a TeX-like syntax"
+  homepage "https://github.com/bartp5/libtexprintf"
+  url "https://github.com/bartp5/libtexprintf/archive/refs/tags/v1.16.tar.gz"
+  sha256 "66f769f7b0fdefadfb205c31dfdbff83e29765578b4900d63ac09f783c701e9f"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/utftex '(\\frac{hello}{world})'"
```

Comment:
```suggestion
    system bin/"utftex", "'(\\frac{hello}{world})'"
```

Reply:
Did this not work @larseggert ?

Reply:
Sorry. I must have reverted this by accident. Will make the change again.